RedHat-UX/red-hat-design-system
# docs: widen description column on code docs table

## What I did

Made the "Description" column on {Element} > Code > API docs table wider.

For a better understanding of what I did, see #1706. A picture (or two) is worth a thousand words.

## Testing Instructions

1. Visit any element's Code page.
2. Scroll down to the API docs table (for example, [Alert](https://deploy-preview-1730--red-hat-design-system.netlify.app/elements/alert/code/#rh-alert))
3. Open a panel with lots of columns (eg: "Attributes")
4. Notice how the "Description" column is now much wider and more readable. 

## Notes to Reviewers

Closes #1706.
